dokka-subprojects/analysis-kotlin-symbols/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/symbols/kdoc/ResolveKDocLink.kt

```diff
@@ -48,11 +52,19 @@ internal fun KaSession.resolveKDocTextLinkToDRI(link: String, context: PsiElemen
  */
 internal fun KaSession.resolveKDocTextLinkToSymbol(link: String, context: PsiElement? = null): KaSymbol? {
     val kDocLink = createKDocLink(link, context)
-    return kDocLink?.let { resolveToSymbol(it) }
+    return kDocLink?.let { analyze(kDocLink) { resolveToSymbol(it) } }
```

Comment:
Same question here regarding additional `analyze` call as for the `resolveKDocLinkToDRI`